bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/zk/ZKMetadataDriverBase.java

```diff
@@ -60,8 +60,12 @@
 
     protected static final String SCHEME = "zk";
 
-    public static String getZKServersFromServiceUri(URI uri) {
-        return uri.getAuthority().replace(";", ",");
+    public static String getZKServersFromServiceUri(URI uri) throws IllegalArgumentException {
```

Comment:
This is great @nicoloboschi!
Can we also get a testcase for this?
Also `getZKServersFromServiceUri` is used in multiple places - can we also account for those usages with respective `catch()` blocks for `IllegalArugmentException`?
Otherwise we'll end up with an annoying stack trace.

Reply:
Thanks @Ghatage 
I added a test for bookie initialization as I think is the only place where it is worth to handle with a checked exception